extensions/doc/makeLuaDocs.lua

```diff
@@ -195,7 +195,8 @@ print("++ Parsing manual...")
 
 -- Get manual version first -- first pass we ignore keys that aren't numbers
 
-    local a, b, k = 1, 0, ""
+    local a = 1
+    local b, k
```

Comment:
This seems potentially concerning.

Reply:
`b` and `k` are redefined straight away [here](https://github.com/cmsj/hammerspoon/blob/d5c32b2799b534ae355cfeb3615831f052cc1f42/extensions/doc/makeLuaDocs.lua#L203):

```lua
a, b, k = LuaManual:find("[\r\n][^\r\n]*<[aA][^\r\n]*%s+[nN][aA][mM][eE]%s*=%s*\"([%d%.]+)\"[^\r\n]*>", a)
```

Therefore, their original value is never used.